### ISSUE TITLE
feature request: be more verbose about connection problems

### ISSUE THREAD

**Issue by pkoevesdi** [martinstoeckli/SilentNotes#174]
Since I have problems connecting to my nextcloud and an Apache WebDAV server, I'm a little bit left stranded with nothing more than the message "couldn't synchnonize, please try again later".
I suggest to give more information on demand, i.e. by clicking something to open a more verbose message or by getting pointed to a log file.
In case, there is already some log, then consider this issue a lack in the documentation. ;-)

**Comment by martinstoeckli** [comment]
You are right, the error message is very generic and there is no log implemented. The log is not used on purpose, to avoid leaking of sensitive information.

It is a balancing act between showing too much details of an exception, which are usually not understood by users and can contain sensitive information, and showing a too generic message. Maybe I should invest more time in generating specific localized error messages. I will keep this open, thank you for the suggestion!
